### PR TITLE
Link to the official FreeBSD manual about BPF

### PR DESCRIPTION
**Pull request by 0mp**
The old link is dead anyway.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ If you are new to eBPF, you may want to try the links described as "introduction
 ### cBPF
 
 - [The BSD Packet Filter: A New Architecture for User-level Packet Capture](http://www.tcpdump.org/papers/bpf-usenix93.pdf) - The original paper about (classic) BPF.
-- [The FreeBSD manual page about BPF](http://www.gsp.com/cgi-bin/man.cgi?topic=bpf)
+- [The FreeBSD manual page about BPF](https://www.freebsd.org/cgi/man.cgi?query=bpf&sektion=&n=1)
```
Comment (qmonnet) [inline readme.md:149]:
```suggestion
- [The FreeBSD manual page about BPF](https://www.freebsd.org/cgi/man.cgi?query=bpf)
```

**Review by qmonnet — CHANGES_REQUESTED** [review]
Thanks for the contribution! Would you mind trimming the URL from the unnecessary bits, please?

**Comment by 0mp** [comment]
Great idea! I have actually specified the "sektion" in the URL now to 4 in order to be sure that the correct manual page is found.

**Review by qmonnet — APPROVED** [review]
Great, thank you!

**Pull request merged**